plugins/janus_sipre.c

```diff
@@ -157,6 +157,8 @@ static struct janus_json_parameter register_parameters[] = {
 	{"username", JANUS_JSON_STRING, JANUS_JSON_PARAM_REQUIRED},
 	{"secret", JANUS_JSON_STRING, 0},
 	{"authuser", JANUS_JSON_STRING, 0},
+	{"display_name", JSON_STRING, 0},
+        {"user_agent", JSON_STRING, 0}
```

Comment:
Indentation is broken here (should be tabs).

Reply:
There was a missing comma, too. Fixed.